tempesta_fw/http_parser.c

```diff
@@ -907,19 +931,25 @@ __parse_connection(TfwHttpMsg *hm, unsigned char *data, size_t len)
 	 * it could be names of any headers, including custom headers.
 	 */
 	__FSM_STATE(I_ConnOther) {
-		__FSM_I_MATCH_MOVE(token, I_ConnOther);
+		msg->flags |= TFW_HTTP_CONN_EXTRA;
```

Comment:
This statement will be executed each time another chunk is seen in the input, even it it's just a single byte. Perhaps, an extra state is justified here that designates the start of a token.